include/cucumber-cpp/internal/RegistrationMacros.hpp

```diff
@@ -21,11 +21,18 @@
 // **************                 CUKE OBJECTS                 ************** //
 // ************************************************************************** //
 
+#if defined(CUKE_ENABLE_BREAKPOINT_SUPPORT)
```

Comment:
I was anticipating that this would make merging it into the public cucumber-cpp easier.